src/seeds.settings.cpp

```diff
@@ -331,6 +331,7 @@ void settings::reset() {
   confwithdesc(name("gratz1.gen"), 100 * 10000, "Base quantity of gratitude tokens received for residents per cycle", medium_impact);
   confwithdesc(name("gratz2.gen"), 200 * 10000, "Base quantity of gratitude tokens received for citizens per cycle", medium_impact);
   confwithdesc(name("gratz.acks"), 10, "Minumum number of gratitude acks to fully use your tokens", medium_impact);
+  confwithdesc(name("gratz.potkp"), 0, "Percent to keep on Gratitude Pot at new round", medium_impact);
```

Comment:
if this is configurable, there needs to be a unit test that tests for this being != 0

Reply:
So this is my change request, the other things are all just comments

Reply:
Added the new test, as requested.
The changes from stitr2 where not possible because `rbegin()` returns a `const` iterator which doesn't allow call to `modify()`, I tried that before. :)